[
   {
      "author_association" : "MEMBER",
      "body" : "I have been able to reproduce the issue of bitcoind processes living on after tests exit in a couple ways:\r\n * #12375: running wallet_encryption.py on macos fails and leaves a bitcoind running\r\n * the feature_dbcrash.py failure that should be fixed in #12902 does the same.\r\n\r\nThis PR causes all nodes to exit in both those situations.",
      "created_at" : "2018-04-06T15:13:58Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/12904#issuecomment-379284768",
      "id" : 379284768,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/12904",
      "updated_at" : "2018-04-06T15:13:58Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/379284768",
      "user" : {
         "avatar_url" : "https://avatars0.githubusercontent.com/u/7463573?v=4",
         "events_url" : "https://api.github.com/users/sdaftuar/events{/privacy}",
         "followers_url" : "https://api.github.com/users/sdaftuar/followers",
         "following_url" : "https://api.github.com/users/sdaftuar/following{/other_user}",
         "gists_url" : "https://api.github.com/users/sdaftuar/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/sdaftuar",
         "id" : 7463573,
         "login" : "sdaftuar",
         "organizations_url" : "https://api.github.com/users/sdaftuar/orgs",
         "received_events_url" : "https://api.github.com/users/sdaftuar/received_events",
         "repos_url" : "https://api.github.com/users/sdaftuar/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/sdaftuar/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/sdaftuar/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/sdaftuar"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "utACK 9caecce6c7b00d3cc43eb220e91e9d1ac42251e8, makes sense",
      "created_at" : "2018-04-06T15:30:22Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/12904#issuecomment-379289688",
      "id" : 379289688,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/12904",
      "updated_at" : "2018-04-06T15:30:22Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/379289688",
      "user" : {
         "avatar_url" : "https://avatars0.githubusercontent.com/u/6399679?v=4",
         "events_url" : "https://api.github.com/users/MarcoFalke/events{/privacy}",
         "followers_url" : "https://api.github.com/users/MarcoFalke/followers",
         "following_url" : "https://api.github.com/users/MarcoFalke/following{/other_user}",
         "gists_url" : "https://api.github.com/users/MarcoFalke/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/MarcoFalke",
         "id" : 6399679,
         "login" : "MarcoFalke",
         "organizations_url" : "https://api.github.com/users/MarcoFalke/orgs",
         "received_events_url" : "https://api.github.com/users/MarcoFalke/received_events",
         "repos_url" : "https://api.github.com/users/MarcoFalke/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/MarcoFalke/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/MarcoFalke/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/MarcoFalke"
      }
   }
]
